### PR TITLE
Update DeepCopy method of each element.

### PR DESCRIPTION
**Pull request by H1Gdev**
#### What does this implement/fix? Explain your changes.

Implementation of `DeepCopy` method of each element class copied attribute via property.
As a result, default value was set at copy destination.

This PR fixes this issue.

### CODE REVIEW

File: Source/DataTypes/SvgAspectRatio.cs [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
+using Svg.DataTypes;
 
 namespace Svg
 {
     /// <summary>
     /// Description of SvgAspectRatio.
     /// </summary>
     [TypeConverter(typeof(SvgPreserveAspectRatioConverter))]
-    public class SvgAspectRatio
+    public class SvgAspectRatio : ICloneable
```
Comment (H1Gdev) [inline Source/DataTypes/SvgAspectRatio.cs]: Implement `ICloneable` for DeepCopy.

File: Source/SvgElement.cs [hunk 2]
```diff
@@ -968,29 +968,29 @@ ISvgNode ISvgNode.DeepCopy()
 
         public virtual SvgElement DeepCopy<T>() where T : SvgElement, new()
         {
-            var newObj = new T();
-            newObj.ID = this.ID;
```
Comment (mrbean-bremen) [inline Source/SvgElement.cs]: I guess the ID does not need to be copied?

Comment (H1Gdev) [reply Source/SvgElement.cs]: `ID` is an attribute, so it is copied.

Comment (mrbean-bremen) [reply Source/SvgElement.cs]: Ah yes, thanks!

**Pull request merged**